code/datums/mind.dm

```diff
@@ -1172,7 +1172,6 @@
 				log_admin("[key_name(usr)] has devil'ed [current]. The devil has been marked as ascendable.")
 			if("sintouched")
 				if(ishuman(current))
-					ticker.mode.sintouched += src

```

Comment:
Uh this would only add if not already added I believe. Can someone confirm this?